app/styles/common/_button.scss

```diff
@@ -0,0 +1,22 @@
+@import "app/styles/ozaria/ozaria-style-params.scss";
```

Comment:
This file could maybe use a comment explaining what we should do with button-related styles–is the idea to move more button styles here and make the home of many/all button styles eventually? Or should this be named more narrowly?